server/cmd/main.go

```diff
@@ -273,6 +276,32 @@ func main() {
 	connToInstanceTracker := handlers.ConnectionToStateMachineInstanceTracker{
 		ConnectToInstanceMap: make(map[uuid.UUID]*machines.StateMachine, 0),
 	}
+	
+		hosts, _, err := regManager.GetRegistrants(&v1alpha1.HostFilter{})
+		if err != nil {
+			log.Error(err)
+		}
+		for _, host := range hosts {
+			log.Infof("For Registrant %s Successfully imported %d models %d components %d relationships %d policy", host.Hostname, host.Summary.Models, host.Summary.Components, host.Summary.Relationships, host.Summary.Policies)
+			nonRegisteredEntityCount := meshmodel.NonImportModel[host.Hostname]
+			if nonRegisteredEntityCount.Models > 0 || nonRegisteredEntityCount.Components > 0 || nonRegisteredEntityCount.Relationships > 0 || nonRegisteredEntityCount.Policies > 0 {
+				log.Errorf("For Registrant %s Failed to import %d models %d components %d relationships %d policy", host.Hostname, nonRegisteredEntityCount.Models, nonRegisteredEntityCount.Components, nonRegisteredEntityCount.Relationships, nonRegisteredEntityCount.Policies)
```

Comment:
```suggestion
				log.Errorf("Registrant %s: Failed to import %d models, %d components, %d relationships, and %d policies", host.Hostname, nonRegisteredEntityCount.Models, nonRegisteredEntityCount.Components, nonRegisteredEntityCount.Relationships, nonRegisteredEntityCount.Policies)
```

Reply:
I'll do the changes as soon as I reach home